### ISSUE TITLE
Android App V3 doesn't log in

### ISSUE THREAD

**Issue by cpranzl** [nextcloud/android#2147]
### Actual behaviour
Asks for login details, you enter them. Nothing happens!

### Expected behaviour
Login once and use
 
### Steps to reproduce
1. Enter user name
2. Enter Password
3. Wait forever (or at least 90 minutes)

### Environment data
Android version: /.0

Device model: Motorola G5

Stock or customized system: Stock

Nextcloud app version: 3

Nextcloud server version: 13.0.0

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by xXSTrikeXx** [comment]
Already exist at #2143

**Comment by xXSTrikeXx** [comment]
Closed, because of duplicate of #2133.
